ansible/ncn_sysctl.yml

```diff
@@ -21,17 +22,21 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
----
-dependencies: []
+# This is a top level play for all NCNs that incorporates all 3 kinds of
+# NCN (management, storage, worker) as well as their corresponding images.
+# The play runs csm.ncn.sysctl which sets kernel parameters
+- hosts: "Management_Master:Management_Worker:Management_Storage:!cfs_image"
+  any_errors_fatal: true
+  remote_user: root
+  roles:
+    - role: csm.ncn.sysctl
+      vars:
+        sysctl_set: true
 
-galaxy_info:
-  role_name: csm.password
-  author: randy.kleinman@hpe.com
-  description: Password management for CSM users
-  company: "Hewlett Packard Enterprise Development LP"
-  license: "MIT"
-  min_ansible_version: 2.9
-  platforms: []
-  galaxy_tags:
-    - system
-    - hpc
\ No newline at end of file
+- hosts: '&cfs_image'
```

Comment:
This needs to be `hosts: "Management_Master:Management_Worker:Management_Storage:&cfs_image"`

See: https://github.com/Cray-HPE/docs-csm/blob/release/1.4/operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md